packages/spanner/system-test/spanner.js

```diff
@@ -1105,7 +1088,9 @@ var spanner = new Spanner(env);
           });
 
           if (attempts < 2) {
```

Comment:
I dont undertstand why was this test passing so far, if transaction id was not being specified?